App/views.py

```diff
@@ -0,0 +1,66 @@
+import json
+import requests
+import urllib.parse
+
+from django.http import HttpResponse
+from django.shortcuts import render
+
+
+def get_recent_english_edits(request):
+    """
+    Display the recent english edits of the user.
+    """
+
+    if request.method == 'POST':
+
+        # Here, we get the username
+        username = request.POST['username']
+        # if username is submitted blank 
+        if not username:
+            return render(request, "App/index.html", {"error": "Please enter a username"})
+
+        # if username is not blank
+        params = {'ucuser': username}
```

Comment:
It would be more readable to put the constant parameters here as well.

Reply:
Done.